src/app/pages/details/resource-box/resource-boxes.jsx

```diff
@@ -16,7 +16,7 @@ function CommonsHubBox({model}) {
                 <br />
                 <a href={model.url}>
                     {model.cta}{' '}
-                    <i className="fa fa-external-link-alt"></i>
+                    <FontAwesomeIcon icon="external-link-alt" />
```

Comment:
oooh, is this a react thing? never seen this before!

Reply:
Yes, it's a React interface for FontAwesome.